doc/user_doc/vic_installation/vch_installer_options.md

```diff
@@ -47,178 +53,263 @@ If you are deploying a virtual container host on vCenter Server, specify a usern
-- Do not use the `bridge-network` distributed port group as the target for any of the other `vic-machine create` networking options.
+<pre>--thumbprint <i>certificate_thumbprint</i></pre>
 
-If you specify an invalid network name, `vic-machine create` fails and suggests valid networks.
+<a name="security"></a>
+## Security Options ##
 
-The `bridge-network` option is **optional** when you are deploying a virtual container host to an ESXi host with no vCenter Server. In this case, if you do not specify `bridge-network`, `vic-machine` creates a  virtual switch and a port group that each have the same name as the virtual container host. You can optionally specify this option to assign an existing port group for use as the bridge network for container VMs. You can also optionally specify this option to create a new virtual switch and port group that have a different name to the virtual container host.
+When you deploy a virtual container host, you specify the type of authentication to use when Docker clients connect to that virtual container host. 
 
-<pre>--bridge-network <i>distributed_port_group_name</i></pre>
+- Two-way authentication with trusted auto-generated TLS certificates that are signed by a Certificate Authority (CA). Specify the [`tls-cname`](#tls-cname) option when you deploy the virtual container host.
+- Server-side authentication with auto-generated, untrusted TLS certificates that are not signed by a CA, with no client-side verification. Specify the [`no-tlsverify`](#no-tlsverify) option when you deploy the virtual container host.
+- Authentication with trusted custom TLS certificates that are signed by a CA.  Specify the [`cert`](#cert) and [`key`](#key) advanced options when you deploy the virtual container host.
+- No TLS authentication. Any Docker client can connect to the virtual container host. Specify the [`no-tls`](#no-tls) advanced option when you deploy the virtual container host.
 
-Wrap the distributed port group name in single quotes (Linux or Mac OS) or double quotes (Windows) if it includes spaces.
+For more information about the possible security configurations for virtual container hosts, see [Securing Virtual Container Host Connections](security.md).
 
-<pre>--bridge-network '<i>distributed port group name</i>'</pre>
+**IMPORTANT**: If you assign a static IP address to a virtual container host on the client network by setting the `--client-network-ip` and `--client-network-gateway` options, you do not need to specify any authentication options. Otherwise, it is **mandatory** to specify an authentication option when you deploy a virtual container host. For information about setting a static IP address on a virtual container host, see [Options for Specifying a Static IP Address for the Virtual Container Host Endpoint VM](#static-ip) in Advanced Options.
```

Comment:
If providing a static client IP and no explicit TLS options, it behaves as if `--tls-cname=<ipaddress>` has been provide.

> Different from not _needing_ to specify options.